packages-content-model/roosterjs-content-model-editor/lib/editor/corePlugins/ContentModelCopyPastePlugin.ts

```diff
@@ -230,9 +230,7 @@ function selectionExToRange(
     let newRange: Range | null = null;
     if (selection.type === SelectionRangeTypes.TableSelection && selection.coordinates) {
         const table = tempDiv.querySelector(`#${selection.table.id}`) as HTMLTableElement;
-        const elementToSelect =
-            table.parentElement?.childElementCount == 1 ? table.parentElement : table;
-        newRange = createRange(elementToSelect);
+        newRange = createRange(table);
```

Comment:
I remember there used to be a bug that when copy table, `<TABLE>` tag is not copied sometimes, and that is why we add a DIV around the table on purpose. Will this make the bug back?

Reply:
I see, I vaguely remember now that you mention it. 
I may need to rethink this then.